airbyte-integrations/connectors/destination-redshift/src/main/java/io/airbyte/integrations/destination/redshift/RedshiftSqlOperations.java

```
@@ -64,4 +96,82 @@ public boolean isValidData(final JsonNode data) {
+    LOGGER.info("Executing operations for Redshift Destination DB engine...");
+    List<String> schemaAndTableWithNotSuperType = writeConfigSet
+        .stream()
+        .flatMap(schemaName -> discoverNotSuperTables(database, schemaName).stream())
+        .toList();
+    if (!schemaAndTableWithNotSuperType.isEmpty()) {
+      updateVarcharDataColumnToSuperDataColumn(database, schemaAndTableWithNotSuperType);
+    }
+    LOGGER.info("Executing operations for Redshift Destination DB engine completed.");
+  }
+
+  /**
+   * @param database   - Database object for interacting with a JDBC connection.
+   * @param schemaName - schema to update.
+   */
+  private List<String> discoverNotSuperTables(final JdbcDatabase database,
+      final String schemaName) {
+    List<String> schemaAndTableWithNotSuperType = new ArrayList<>();
+    try {
+      LOGGER.info("Discovering NOT SUPER table types...");
+      database.execute(String.format("set search_path to %s", schemaName));
+      final List<JsonNode> tablesNameWithoutSuperDatatype = database.bufferedResultSetQuery(
+          conn -> conn.createStatement().executeQuery(String.format(SELECT_ALL_TABLES_WITH_NOT_SUPER_TYPE_SQL_STATEMENT,
+              schemaName,
+              JavaBaseConstants.COLUMN_NAME_DATA,
+              JavaBaseConstants.COLUMN_NAME_EMITTED_AT,
+              JavaBaseConstants.COLUMN_NAME_AB_ID)),
+          getDefaultSourceOperations()::rowToJson);
+      if (tablesNameWithoutSuperDatatype.isEmpty()) {
+        return Collections.emptyList();
+      } else {
+        tablesNameWithoutSuperDatatype.forEach(e ->
+            schemaAndTableWithNotSuperType.add(e.get("schemaname").textValue() + "." + e.get("tablename").textValue()));
+        return schemaAndTableWithNotSuperType;
+      }
+    } catch (SQLException e) {
+      LOGGER.error("Error during discoverNotSuperTables() appears: ", e);
```

Why are we swallowing the exception and doing nothing